README.md

```diff
@@ -85,9 +96,11 @@ Journaling provides a number of different configuation options that can be set i
 
 #### `Journaled.job_priority` (default: 20)
 
-  This can be used to configure what `priority` the Delayed Jobs are enqueued with. This will be applied to all the Journaled::Devivery jobs that are created by this application.
+  This can be used to configure what `priority` the ActiveJobs are enqueued with. This will be applied to all the Journaled::DeliveryJobs that are created by this application.
```

Comment:
```suggestion
  This can be used to configure what `priority` the ActiveJobs are enqueued with. This will be applied to all the `Journaled::DeliveryJob`s that are created by this application.
```